src/components/WwwFrame/PromotionalBanner/GlobalPromotionalBannerContentful.vue

```diff
@@ -136,9 +170,9 @@ export default {
 				'/possibility/12-days-of-lending',
 				'/possibility/year-end'
 			];
-			// First check if Appeal Banner or Appeal Banner Matching
+			// First check if Appeal Banner
 			// is active and the user is not on a blacklisted page URL
-			if ((this.appealEnabled || this.appealMatchEnabled) && !blacklist.includes(this.$route.path)) {
```

Comment:
appealMatchedEnabled is referenced here but nowhere else in the component, it is unused